### PR TITLE
I almost completed the dashboard component and as well styling parts.

### PR DESCRIPTION
**Pull request by Amaya-2001**

### CODE REVIEW

File: src/app/components/login/login.component.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  ngOnInit(): void {
+    this.loginForm = this.fb.group({
+      email:['', [Validators.required, Validators.email]],
+      password:['', Validators.required]
+    })
+    
+  }
+  onLogin(){
+    console.log(this.loginForm.value);
```
Comment (sujeewasandeepa) [inline src/app/components/login/login.component.ts]: Remove console logs

File: src/app/components/login/login.component.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    this.auth.login(this.loginForm.value).subscribe({
+      next:(res)=>{
+        this.toast.success({detail:"Success!",summary:res.message,duration:5000})
+        this.loginForm.reset();
+        this.router.navigate(['grama-niladhari/home'])
+      },
```
Comment (sujeewasandeepa) [inline src/app/components/login/login.component.ts]: Check code formatting and indentation

File: src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
+h5{
+    font-size: 20px;
+    color: rgb(0,83,78);
+  }  
+
+.appoinment-content{
+    overflow-y: scroll;
```
Comment (sujeewasandeepa) [inline src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.css:7]: In the UI there is no scroll bar

File: src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-variable */
```
Comment (sujeewasandeepa) [inline src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.spec.ts:1]: don't disable warnings

File: src/app/grama-niladhari/components/dashboard/daily-feed/daily-feed.component.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  span{
+    margin-left: 10px;
+  }
+
+  hr{
+    color: rgb(0,83,78);
+  }
+  .card-body{
+   
+
```
Comment (sujeewasandeepa) [inline src/app/grama-niladhari/components/dashboard/daily-feed/daily-feed.component.css]: check code formatting of the whole file

File: src/app/grama-niladhari/components/dashboard/daily-feed/daily-feed.component.html
Comment (sujeewasandeepa) [inline src/app/grama-niladhari/components/dashboard/daily-feed/daily-feed.component.html:1]: Fix formatting

File: src/app/grama-niladhari/components/dashboard/gn-dashboard/gn-dashboard/gn-dashboard.component.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-6">
+        <app-daily-feed></app-daily-feed>
+      </div>
+      <div class="col-sm-4">
+        <app-calender></app-calender>
+        <app-appoinments></app-appoinments>
+      </div>
+      </div>
```
Comment (sujeewasandeepa) [inline src/app/grama-niladhari/components/dashboard/gn-dashboard/gn-dashboard/gn-dashboard.component.html:16]: formatting

File: src/app/grama-niladhari/components/onboarding/gn-onboarding/gn-onboarding.component.ts [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+})
+export class GnOnboardingComponent implements OnInit {
+
+  onboardForm!:FormGroup
+
+  constructor(private fb:FormBuilder, private auth:AuthService, private toast:NgToastService) {}
+  ngOnInit(): void {
+    this.onboardForm =  this.fb.group({
+      GN_ID:['', Validators.required],
+      //divisionId:['', Validators.required],
```
Comment (sujeewasandeepa) [inline src/app/grama-niladhari/components/onboarding/gn-onboarding/gn-onboarding.component.ts]: remove un wanted comments

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,25 @@
...
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>SLHAngularJSWeb</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
```
Comment (sujeewasandeepa) [inline src/index.html:10]: Is this a secret key? If it is something sensitive put inside an env file

**Review by sujeewasandeepa — CHANGES_REQUESTED** [review]

File: src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.css (hunk 3 above)
Comment (Amaya-2001) [reply src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.css:7]: I did not get you. I used the scroll bar in here
Screenshot 2024-01-23 073741

File: src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.spec.ts (hunk 4 above)
Comment (Amaya-2001) [reply src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.spec.ts:1]: It was automatically commented. so what should I do in my side? When I uncommented, it shows an error.
image

File: src/index.html (hunk 8 above)
Comment (Amaya-2001) [reply src/index.html:10]: No. It's not a secret key , I used it to import bootstrap css package as CDN

File: src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.css (hunk 3 above)
Comment (sujeewasandeepa) [reply src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.css:7]: I mean in the figma design there is no scroll bar, are we going to add one?

**Review by sujeewasandeepa — APPROVED** [review]

File: src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.spec.ts (hunk 4 above)
Comment (sujeewasandeepa) [reply src/app/grama-niladhari/components/dashboard/appoinments/appoinments.component.spec.ts:1]: Huh okay ignore it then leave it commented

**Pull request merged**